src/extras.py

```diff
@@ -37,7 +37,13 @@ def _augment_with_salary_cached(last: Optional[str], first: Optional[str]) -> st
     if not results:
         return ""
     s = results[0]
-    projected = Decimal(s["hourly_rate"]) * 40 * 50
+    hourly_rate = s["hourly_rate"]
+    # City of Seattle wage data formats rate as "$    <number>" for some
+    # horrible reason, so we clean this data a bit for decimal conversion.
+    if isinstance(hourly_rate, str):
+        hourly_rate = hourly_rate.replace("$", "").replace(" ", "")
```

Comment:
You can use `hourly_rate.trim(“$ “)`, if you want, unless they put spaces in the middle of numbers.

Reply:
Hmm, I'm not sure Python has a `str::trim` method :open_mouth: However, I can use `strip("$ ")` and I think that'll have the same effect! Thanks!